java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -205,7 +207,7 @@ export function activate(context: ExtensionContext) {
 
         let debugDescriptionFactory = new NetBeansDebugAdapterDescriptionFactory();
         context.subscriptions.push(vscode.debug.registerDebugAdapterDescriptorFactory('java-polyglot', debugDescriptionFactory));
-        window.showInformationMessage('Java Polyglot Debug Adapter ready.');
+        window.setStatusBarMessage('Java Polyglot Debug Adapter ready.', 5000);
```

Comment:
We don't really need this message at all. It is a heritage from the early attempts to make the Debug Adapter Protocol work.